README.md

```diff
@@ -39,3 +39,24 @@ When we took on the task of importing the standards, a few principles guided our
 * **Easy to fix**: Converting 50 state standards is a monumental task (and one we're grateful ASN takes on). There will be typos and we want users to find them and fix them. Thus, each set of standards is versioned and changes can be submitted for approval.
 
 * **Easy to add to**: Schools & districts create their own standards (outside of their state). They should be able to publish those standards and share them with their vendors.
+
+## Development
+
+There are a few tasks that must be completed to get running with a local copy of the project for development purposes:
+
+* Sign up with [Algolia](https://www.algolia.com) and obtain your Application ID and Admin API Key.
+* Set the Algolia keys in your env: `export ALGOLIA_APPLICATION_ID=[APPID]; export  ALGOLIA_API_KEY=[KEYID]` (to make this persistent, set in your bash profile)
+* Install [MongoDB](https://www.mongodb.org/)
+* Run `mongorestore sources/jurisdiction-mongodb-dump` to do an initial import of jurisdictions
+* Clone the repo and run `bundle install` to get all the gems you will need
+* Run `bundle exec ruby importer/importer.rb` to import all the standards data into your local mongo (this requires internet access, and will take a while, go get a snack)
+* In the mongo console, you will need to create the api docs user:
+
+```
+> use standards;
+> db.createCollection('users')
+> db.users.insert({profile: true, email: 'user@user.com', apiKey: 'vZKoJwFB1PTJnozKBSANADc3'})
+```
+
+* Run `foreman s` to start the application on your local machine
```

Comment:
People will want to run `foreman start dev` in development so the server reloads on file changes.